src/cli.js

```diff
@@ -171,6 +166,8 @@ Options
   --repl                                   Open the REPL (drive Smashtest from command line) (-r)
   --report-domain=<domain>                 Domain and port where report server should run (domain or domain:port format)
   --report-server=<true/false>             Whether to run a server during run for live report updates
+  --report-path=<path>/<domain>            Set absolute path, domain or domain:port for change report location
```

Comment:
Improved wording: "Set to absolute path, domain, or domain:port to set report location"